tests/lib/cli.js

```diff
@@ -358,6 +358,28 @@ describe("cli", function() {
     });
 
 
+    describe("when given patterns to ignore", function() {
```

Comment:
See https://github.com/eslint/eslint/blob/master/tests/lib/cli.js#L647 for a better way to test (note https://github.com/eslint/eslint/blob/master/tests/lib/cli.js#L636)